cmd/server/app/migrate_up.go

```diff
@@ -127,10 +130,64 @@ var upCmd = &cobra.Command{
 			return fmt.Errorf("error preparing authz client: %w", err)
 		}
 
-		return nil
+		store := db.NewStore(dbConn)
+		return migrateProjectsRootAndMetadata(ctx, cmd, store)
 	},
 }
 
+func migrateProjectsRootAndMetadata(ctx context.Context, cmd *cobra.Command, store db.Store) error {
```

Comment:
What happens with OpenFGA authorizations here?  It seems like it's worth a comment explaining whether there's any migration / cleanup, and if not, why not.

Reply:
What do you mean? We never had openfga relationships on the org projects we removed.

Reply:
Let's memorialize that in a comment, is what I'm suggesting then.  (I suspected that was true from having poked at the data, but it's a little bit of an odd choice.)